test/suites/dev-tanssi/xcm-token-reception/test_dmp_token_reception _appendix.ts

```diff
@@ -0,0 +1,107 @@
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
```

Comment:
Some of these files have a space in its filename, can you rename them?

Reply:
yes!